smarts/core/traffic_history_provider.py

```diff
@@ -96,8 +96,22 @@ def step(self, provider_actions, dt, elapsed_sim_time) -> ProviderState:
                         [*vehicle_state["position"][:2], 0,],
                         Heading(vehicle_state["heading"]),
                     ),
-                    # TODO: specify dimensions
-                    dimensions=VEHICLE_CONFIGS[vehicle_type].dimensions,
+                    dimensions=BoundingBox(
+                        length=vehicle_state.get(
+                            "vehicle_length",
+                            VEHICLE_CONFIGS[vehicle_type].dimensions.length,
+                        ),
+                        width=vehicle_state.get(
+                            "vehicle_width",
+                            VEHICLE_CONFIGS[vehicle_type].dimensions.width,
+                        ),
+                        # Note: Neither NGSIM nor INTERACTION provide
+                        #       the height of the vehicles.
+                        height=vehicle_state.get(
+                            "vehicle_height",
+                            VEHICLE_CONFIGS[vehicle_type].dimensions.height,
```

Comment:
The one thing I can question is the usefulness of the defaults here. Perhaps it might be better to just assert.

Reply:
But `height` is not provided in the dataset. So for the length, width, height, I wanted the assigning to be consistent, if not provided, use a default value.